src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/Option.js

```diff
@@ -110,7 +102,6 @@ export default class Option<T> extends React.PureComponent<Props<T>> {
                     className={optionClass}
                     disabled={disabled}
                     onClick={this.handleButtonClick}
-                    onKeyDown={this.handleButtonKeyDown}
                     ref={this.setButtonRef}
                     style={{minWidth: anchorWidth + ANCHOR_WIDTH_DIFFERENCE}}
```

Comment:
```suggestion
                    style={{minWidth: anchorWidth + ANCHOR_WIDTH_DIFFERENCE}}
                    type="button"
```

Did see this only yet sorry, this button should also define a type property.